src/Serilog.Sinks.ColoredConsole/project.json

```diff
@@ -0,0 +1,29 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The colored console sink for Serilog",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "console", "coloredconsole" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": { "target": "project" }
+  },
+  "compilationOptions": {
+    "keyFile": "../../assets/Serilog.snk"
+  },
+  "frameworks": {
+    "net45": {
+    },
+    "dotnet5.1": {
+      "dependencies": {
+        "System.Console": "4.0.0-beta-23516"
+      }
+    },
+    "dotnet5.4": {
```

Comment:
This doesn't need `dotnet5.4`. If it supports `dotnet5.1`, it will also be compatible with `dotnet5.1` (that's the beauty of this, compared to PCLs) :smile:

The only reason to multi-target is if some features are only available for one target.